pkg/controller/container-runtime-config/helpers.go

```diff
@@ -58,17 +61,21 @@ type tomlConfigCRIO struct {
 
 type updateConfigFunc func(data []byte, internal *mcfgv1.ContainerRuntimeConfiguration) ([]byte, error)
 
-func createNewCtrRuntimeConfigIgnition(storageTOMLConfig, crioTOMLConfig []byte) igntypes.Config {
+// createNewIgnition returns an ignition config containing the new data configure by the CRs
```

Comment:
This function does not see any CRs. It would be nice to describe what are the keys and values in the `configs` map (especially if the `nil` semantics will remain).

Reply:
Fixed